pysass.cpp

```diff
@@ -535,6 +540,18 @@ PySass_compile_string(PyObject *self, PyObject *args) {
     sass_option_set_include_path(options, include_paths);
     sass_option_set_precision(options, precision);
     sass_option_set_is_indented_syntax_src(options, indented);
+    sass_option_set_source_map_contents(options, source_map_contents);
+    sass_option_set_source_map_embed(options, source_map_embed);
+    sass_option_set_omit_source_map_url(options, omit_source_map_url);
+
+    if (PyBytes_Check(source_map_root)) {
+        if (PyBytes_GET_SIZE(source_map_root)) {
+            sass_option_set_source_map_root(
+                options, PyBytes_AS_STRING(source_map_root)
```

Comment:
does libsass copy this value or do we need to dupe our value? (same question for the calls above as well) -- we've had a few issues in the past with fuzzy ownership of `char*` literals

Reply:
seems we don't have to -- `libsass` will [copy the cstring for us](https://github.com/sass/libsass/blob/664a9c2a08fefb2b41856a4f76a60b2ea83f02c7/src/sass_context.cpp#L283-L285)